x-pack/plugins/snapshot_restore/__jest__/client_integration/home.test.ts

```diff
@@ -748,7 +748,7 @@ describe('<SnapshotRestoreHome />', () => {
               const { find } = testBed;
 
               const tabs = find('snapshotDetail.tab');
-              const selectedTab = find('snapshotDetail').find('.euiTab-isSelected');
+              const selectedTab = find('snapshotDetail').find('.euiTab-isSelected').last();
```

Comment:
Can you explain why the `.last()` is needed now?

Reply:
Because there's an infinite ways to do anything in JS...

Enzyme's [`hostNodes`](https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/hostNodes.html) can be a good alternative in these scenarios where you need to trim down to only "real" DOM elements. Discovered this method during the last upgrade, until now I've also been using `last()`.